app/controllers/api/v1/Web.java

```diff
@@ -0,0 +1,1274 @@
+/*
+ * Copyright 2016 LinkedIn Corp.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+package controllers.api.v1;
+
+import com.google.common.collect.Lists;
+import com.google.gson.Gson;
+import com.google.gson.JsonArray;
+import com.google.gson.JsonObject;
+import com.linkedin.drelephant.analysis.Severity;
+import com.linkedin.drelephant.util.Utils;
+import controllers.ControllerUtil;
+import controllers.IdUrlPair;
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.Comparator;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import models.AppHeuristicResult;
+import models.AppHeuristicResultDetails;
+import models.AppResult;
+import org.apache.log4j.Logger;
+import play.mvc.Controller;
+import play.mvc.Result;
+
+
+/**
+ * The Web controller defines the rest interfaces for the Dr. Elephant User interface.
+ */
+public class Web extends Controller {
+
+  private static final Logger logger = Logger.getLogger(Web.class);
+  private static final int MAX_APPLICATIONS = 50;
+  private static final int MAX_APPLICATIONS_IN_WORKFLOW = 5000;
```

Comment:
I don't think we should add a config for these constants, these should be configurable by the users of the rest interface.  In the subsequent PRs I'll make the rest endpoints configurable and add parameters for these values so that the applications using the rest end point can decide on these numbers.